e2e-tests/spring-rest-openapi-v2/src/main/java/com/foo/rest/examples/spring/unauthenticatedswaggeraccess/UnauthenticatedSwaggerApplicationWebSecurityConfig.java

```diff
@@ -0,0 +1,26 @@
+package com.foo.rest.examples.spring.unauthenticatedswaggeraccess;
+
+import org.springframework.context.annotation.Configuration;
+import org.springframework.http.HttpMethod;
+import org.springframework.security.config.annotation.web.builders.HttpSecurity;
+import org.springframework.security.config.annotation.web.configuration.EnableWebSecurity;
+import org.springframework.security.config.annotation.web.configuration.WebSecurityConfigurerAdapter;
+
+@Configuration
+@EnableWebSecurity
+public class UnauthenticatedSwaggerApplicationWebSecurityConfig extends WebSecurityConfigurerAdapter {
+
+    // security configuration
+    protected void configure(HttpSecurity http) throws Exception {
+        http.authorizeRequests()
+                .antMatchers("/v2/api-docs").permitAll()
+                .antMatchers(HttpMethod.GET, "/endpoint1").permitAll()
+                .antMatchers(HttpMethod.PUT, "/endpoint2").hasAnyRole("AUTH_USER")
```

Comment:
same comments as other class

Reply:
Hi Andrea,

Thanks for your comment. I have fixed this one.